product_multi_image/__openerp__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3).
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2014 Serv. Tecnol. Avanzados Pedro M. Baeza
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# © 2016 Pedro M. Baeza, Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Multiple Images in Products",
+    "version": "9.0.1.0.0",
+    "author": "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Antiun Ingeniería, Tecnativa, Sodexis, "
```

Comment:
Ah forget it, I saw 1 per line in v8 and did not realize you mixed some in the same line. My fault.